src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  padding-top: $padding;
+  justify-content: center;
+  align-items: center;
+
+  &__main {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $day-gap;
+    width: 7 * $day-size + $main-padding + 7 * $day-gap;
+    padding: $main-padding;
+
+    @each $day, $number in $days {
+      &--starting-day--#{$day} :nth-child(1) {
+        margin-left: $number * ($day-gap + $day-size);
+      }
+    }
+
+    @for $num from 28 through 31 {
+      &--month-length--#{$num}>.calendar__day:nth-child(n + #{$num + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    @include square($day-size);
+    border: 1px solid #000;
+    background-color: #eee;
+    box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, use `*` in normalize for that

Reply:
done